config/settings.yml

```diff
@@ -79,7 +79,7 @@ google:
   bigquery:
     project_id: replaceme
     dataset: replaceme
-    api_json_key: ""
+    api_json_key: "{}"
```

Comment:
why empty object? can it not be nil?

Reply:
It should be an object, at least that's what the API key is that we use here. I don't remember what broke as a result of it being `nil`, but this is technically More Correct.